lib/sprockets/sassc_compressor.rb

```diff
@@ -0,0 +1,22 @@
+require 'sprockets/autoload'
+require 'sprockets/sassc_importer'
+require 'sprockets/sass_compressor'
+
+module Sprockets
+  class SasscCompressor < SassCompressor
+    def initialize(options = {})
+      @options = {
+        syntax: :scss,
+        cache: false,
+        read_cache: false,
```

Comment:
Based on https://github.com/sass/sassc-ruby/blob/master/lib/sassc/engine.rb I don’t think `cache` or `read_cache` are supported options in `sassc-ruby`.

Based on https://github.com/sass/libsass/blob/713e1628f865ee7f8fb373086a22da1c7160e8fe/include/sass_context.h#L88 it also doesn’t look like these options are available in libsass, so we should probably leave these options out for now.